CRM/Admin/Form/Options.php

```diff
@@ -164,14 +170,20 @@ public function buildQuickForm() {
       CRM_Core_DAO::getAttribute('CRM_Core_DAO_OptionValue', 'label'),
       TRUE
     );
-
+    $domainSpecificOptionGroups = array('from_email_address');
```

Comment:
Looks to me like this variable should be referencing `CRM_Core_OptionGroup::$_domainIDGroups` and not just a hard-coded array.

Reply:
Good point @colemanw